### ISSUE TITLE
Command-line option to control the number of words in the passphrase.

### ISSUE THREAD

**Issue by wfscheper** [wfscheper/xkcdpwd#2]
As a user of xkcdpwd, I would like to control the number of words in the passphrase via a command-line option.

Examples
```shell
$ xkcdpwd -words=5
```
```shell
$ xkcdpwd -words=2
error: cannot generate a passphrase with less than 30 bits of entropy
```

**Comment by jlucktay** [comment]
Hi, @wfscheper!
Found this when [searching for the tag `good first issue`](https://github.com/search?l=Go&o=desc&p=2&q=label%3A%22good+first+issue%22+no%3Aassignee+&s=updated&state=open&type=Issues).
I've created a fork to take a swing at this.
Will PR when I have something!

**Comment by wfscheper** [comment]
Hey @jlucktay, sounds good. The test suite will need some adjusting for this feature. The way I hack in a static dictionary to test command output won't work very well with the minimum entropy check. My plan is to change things so that we check the number of words (4 by default) and the number of passphrases (10 by default) instead of strictly checking the exact output.

**Comment by jlucktay** [comment]
Could we make use of something like [this](https://github.com/nbutton23/zxcvbn-go) to check the password strength?
I ended up there via an [old blog post (2012)](https://blogs.dropbox.com/tech/2012/04/zxcvbn-realistic-password-strength-estimation/) from Dropbox.

**Comment by wfscheper** [comment]
While interesting from a technical point of view, xkcdpwd benefits from knowing how its passphrases are generated (ie. we're picking X words from a dictionary of Y choices with a decent crypto RNG), so using the basic X * log2(Y) calculation for entropy is okay. We're just trying to keep the user form shooting themselves in the foot.

**Comment by wfscheper** [comment]
Closed by #10
